language/resource.language.it_it/strings.po

```diff
@@ -67,7 +67,7 @@ msgstr "Personalizzazione Widget"
 #: /1080i/Includes_Shortcuts_Window.xml
 msgctxt "#31010"
 msgid "Welcome to the Arctic Fuse widget customisation screen!"
-msgstr "Benvenutə nella schermata di personalizzazione dei widget di Arctic Fuse!"
+msgstr "Benvenut@ nella schermata di personalizzazione dei widget di Arctic Fuse!"
```

Comment:
That's actually not a typo.

I used "ə" to express gender neutrality because most italian words have different suffixes when used to address a male or a female, and most expressions must address the interlocutor's gender, so it has become common in recent years to use  the character "ə" in written sentences to avoid that distinction.

I changed to a "@" because "ə" wasn't shown on the interface and that was the next best thing, but I then realized I had changed the font to do some testing and it is actually rendered correctly with your skin default font.

I will just revert the change and go back to the "ə".

I could also use a "/" to write both suffixes as "Benvenuta/o" but the first alternative is better in my opinion.

Reply:
Yeah I think revert back to the correct character. 

The display issue is due to a reported bug with Google's Roboto Font which the skin uses. It looks like it was fixed a while back in the main font.

I'll update the fonts with more recent versions and see if it resolves it.

Reply:
@seppiola - Did you have a chance to try out this commit that updates the fonts? https://github.com/jurialmunkey/skin.arctic.fuse.2/commit/0f1b1f5d64598bb28ce3558eb427dddbabde2af7

I'm wondering if it fixes the issue with the ə